tests/gold_tests/autest-site/init.cli.ext

```diff
@@ -17,21 +17,29 @@
 #  limitations under the License.
 
 import sys
+import microserver
 
 if sys.version_info < (3, 6, 0):
     host.WriteError(
         "You need python 3.6 or later to run these tests\n", show_stack=False)
 
-autest_version = "1.8.1"
-if AuTestVersion() < autest_version:
+needed_autest_version = "1.8.1"
+found_autest_version = AuTestVersion()
+if AuTestVersion() < needed_autest_version:
     host.WriteError(
-        "Tests need AuTest version {needed_version} or better, found version {found_version}\n"
-        "Please update AuTest:\n  pip install --upgrade autest\n".format(
-            needed_version=autest_version,
-            found_version=AuTestVersion()),
+        f"Tests need AuTest version {needed_autest_version} or better, found version {found_autest_version}\n"
+        "Please update AuTest:\n  pipenv --rm && pipenv install\n",
```

Comment:
So my only real concern is that this test is not bound to pipenv. it more of a pip issue. as stated before the message was to update the package. Maybe I being picky but the autest.sh script and autest checks are separate items.

Reply:
Right. I appreciate this.

Maybe the question is whether autest.sh is "official." Alan ran with autest.sh and executed the previous stated commands and they did not help because they did not apply to his pipenv. Since most people will run with autest.sh, I figured this message was more helpful than the previous one.

Should we simply remove any specific command and just indicate that the package needs to be updated?